### DISCUSSION TITLE
Show description on dashboard lists

### DISCUSSION THREAD

**Discussion by yostevie** [grafana/grafana#68694] [Dashboards, Panels & Visualizations]
We have many many dashboards, often with similar names, so, seeing the description in the dashboard / folder view would be a really simple way of helping users identify which dashboard they want.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82689. Feel free to continue the discussion around this there. Thank you!
